src/tokens/transformation/common/leoParser.js

```diff
@@ -30,12 +34,25 @@ module.exports = {
     // export variable references in effects.
     // At the moment, we just whitelist a few color groups.
     const allowedGroups = ['elevation', 'primary', 'secondary']
-    const effectColors = allowedGroups
+    const universalEffectColors = allowedGroups
```

Comment:
Maybe worth extracting this to a function we can reuse for `layerEffectColors` and `universalEffectColors`